dotCMS/src/main/java/com/dotcms/publisher/pusher/wrapper/StructureWrapper.java

```diff
@@ -31,12 +31,12 @@ public void setFieldVariables(List<FieldVariable> fieldVariables) {
         this.fieldVariables = fieldVariables;
     }
 
-	public String getWorkflowSchemaName() {
-        return workflowSchemaName;
+	public List<String> getWorkflowSchemaNames() {
+        return workflowSchemaNames;
     }
 
-    public void setWorkflowSchemaName(String workflowSchemaName) {
-        this.workflowSchemaName = workflowSchemaName;
+    public void setWorkflowSchemaNames(List<String> workflowSchemaNames) {
+        this.workflowSchemaNames = workflowSchemaNames;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "workflowSchemaNames". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)